flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -40,4 +39,11 @@ private FlinkConfigOptions() {
           .intType()
           .defaultValue(100)
           .withDescription("Sets max infer parallelism for source operator.");
+
+  public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_EXPOSE_SPLIT_LOCALITY_INFO =
+      ConfigOptions.key("table.exec.iceberg.expose-split-locality-info")
+          .booleanType()
+          .noDefaultValue()
+          .withDescription("Controls whether to report locality information to Flink while allocating input " +
+              "partitions.");
```

Comment:
I assume this is broken into two lines because the sentence exceeds the checkstyle length limit. I also think we can make the description more concise.

```
Expose locality host information to use Flink's locality aware split assigner.
```

Reply:
ok.